.evergreen/run-typescript.sh

```diff
@@ -5,14 +5,25 @@ source "${PROJECT_DIRECTORY}/.evergreen/init-nvm.sh"
 
 set -o xtrace
 
-function get_current_ts_version {
-    node -e "console.log(require('./package-lock.json').dependencies.typescript.version)"
-}
+if [ -z "$TS_CHECK" ]; then echo "TS_CHECK must be set to either COMPILE_DRIVER or CHECK_TYPES"; exit 1; fi
+if [ -z "$TS_VERSION" ]; then echo "TS_VERSION must be set"; exit 1; fi
 
-CURRENT_TS_VERSION=$(get_current_ts_version)
+if [ ! -f "mongodb.d.ts" ]; then
+    # mongodb.d.ts should always exist because of the installation in prior steps
+    # but in case it doesn't, build it
+    npm i
+fi
+
+function get_ts_version {
```

Comment:
```suggestion
function get_ts_version() {
```
just a preference, up to you

Reply:
nope, I prefer this as well. I dislike that bash has 4 different syntaxes for declaring functions.